config/prow/cluster/monitoring/mixins/grafana_dashboards/tide.jsonnet

```diff
@@ -1,3 +1,5 @@
+# PROW_INSTANCE_SPECIFIC
+# kubernetes/kubernetes repo in this file
```

Comment:
This is addressed separately here: https://github.com/kubernetes/test-infra/pull/19552